bin/broker_submission.py

```diff
@@ -51,7 +53,9 @@ def main():
 
     # Optionally Set the valid VCF and metadata file
     brokering = EloadBrokering(args.eload, args.vcf_files, args.metadata_file)
-    brokering.broker(brokering_tasks_to_force=args.force)
+    if args.report:
```

Comment:
i think this should be flipped, to be consistent with [validation reporting](https://github.com/EBIvariation/eva-submission/blob/master/bin/validate_submission.py#L57)
```suggestion
    if not args.report:
```